lib/libutils/ext/trace.c

```diff
@@ -53,9 +53,8 @@ int trace_get_level(void)
 
 static const char *trace_level_to_string(int level, bool level_ok)
 {
-	static const char lvl_strs[][9] = {
-		"UNKNOWN:", "ERROR:  ", "INFO:   ", "DEBUG:  ",
-		"FLOW:   " };
+	static const char lvl_strs[][2] = {
```

Comment:
This could be an array of chars instead of an array of strings if `"MESSAGE:"` below was exchanged for `'M'`